ee/maintained-apps/outputs/darwin/adobe-acrobat-reader.json

```diff
@@ -1,19 +1,19 @@
+    "1fa6b6bb-a240-4a6a-a880-9cdee36f8e8a": "#!/bin/sh\n\n# variables\nAPPDIR=\"/Applications/\"\nTMPDIR=$(dirname \"$(realpath $INSTALLER_PATH)\")\n\n# extract contents\nMOUNT_POINT=$(mktemp -d /tmp/dmg_mount_XXXXXX)\nhdiutil attach -plist -nobrowse -readonly -mountpoint \"$MOUNT_POINT\" \"$INSTALLER_PATH\"\nsudo cp -R \"$MOUNT_POINT\"/* \"$TMPDIR\"\nhdiutil detach \"$MOUNT_POINT\"\n# install pkg files\nsudo installer -pkg \"$TMPDIR/AcroRdrDC_2400520414_MUI.pkg\" -target /\n",
```

Comment:
Can you double-check how these scripts compare to the FMAv1-ingested ones? They should be identical.

Also, please include a full ingest command run in this commit; I'd expect to see more changes.

Additionally, the intent for output files...and I just missed this apparently...was to do `outputs/[slug].json`, which means that what's currently on `main` is flipped (to be fair, I swapped the dir structure rather late on #26080). Idea here is that if someone is adding or maintaining an individual app, they only have one directory to look at.

Reply:
I spot checked the apps that I have included in this PR and the outputs look correct